infra/Makefile

```
@@ -1,86 +1,37 @@
-			exit 1; \
-		fi; \
-		rm -f test_output.log
-
-# Format code
-fmt:
-	cd .. && go fmt ./...
-
-# Run linter
-lint:
-	cd .. && go vet ./...
+help: ## 🚀 Show this help message
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (11).

(maxbodylength)